app/pages/lab/apply-for-beta-form.jsx

```diff
@@ -8,28 +8,17 @@ const MINIMUM_SUBJECT_COUNT = 100;
 const REQUIRED_PAGES = ['Research', 'FAQ'];
 
 // Static functions
-const projectHasActiveWorkflows = (workflows) => {
-  return workflows.some((workflow) => {
-    return workflow.active;
-  });
+const projectHasActiveWorkflows = (project) => {
+  if (project.links.active_workflows) {
```

Comment:
I think this will be true if `project.links.active_workflows` is an empty array.

Reply:
AFAIK, the `active_workflows` field gets added to `links` only if its `length > 0`.

Reply:
In that case, could you explicitly check for `project.links.active_workflows.length > 0` here?

Reply:
Wouldn't that throw an error when `active_workflows` is `undefined`?

Reply:
Then check for `project.links.active_workflows && project.links.active_workflows.length > 0`.